src/Adapters/Files/GridStoreAdapter.js

```diff
@@ -110,6 +110,10 @@ export class GridStoreAdapter extends FilesAdapter {
     }
     return this._client.close(false);
   }
+
+  validateFilename(filename) {
+    return validateFilename(filename);
```

Comment:
Can you write a test case for this one too? This will help improve code coverage

Reply:
Done